controllers/users.js

```diff
@@ -142,12 +142,26 @@ const postUserPicture = async (req, res) => {
     return res.boom.badImplementation('An internal server error occurred')
   }
 }
-
+const identityURL = async (req, res) => {
+  try {
+    const userId = req.userData.id
+    await userQuery.addOrUpdate(req.body, userId)
+    const { user } = await userQuery.fetchUser({ userId })
```

Comment:
Not getting an error from `userQuery.addOrUpdate` itself means the value has been updated. We dont need to get the values from DB again here.